src/views/Maps.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <div class="map">
```

Comment:
We forgot to rename this class, It should be "maps". After that fix the styles too.